<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
	qproperty-alignment: 'AlignVCenter | AlignRight'; 
	border: 1px solid gray; 
	background-color: white;
	color: black;
} 

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(215, 215, 215); 
	border: 1px solid gray; 
	color: black;
}
 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7); 
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus_minus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(215, 215, 215); 
	border: 1px solid gray; 
	color: black;
}
 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7); 
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(215, 215, 215); 
	border: 1px solid gray; 
	color: black;
}
 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7); 
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 151, 57); 
	color: white; 
	border: 1px solid gray; } 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739); 
} </string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seven">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eight">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 151, 57); 
	color: white; 
	border: 1px solid gray; } 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739); 
} </string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_four">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_six">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_five">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 151, 57); 
	color: white; 
	border: 1px solid gray; } 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739); 
} </string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_three">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_two">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 151, 57); 
	color: white; 
	border: 1px solid gray; } 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739); 
} </string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dec">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(215, 215, 215); 
	border: 1px solid gray; 
	color: black;
}
 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7); 
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: rgb(255, 151, 57); 
	color: white; 
	border: 1px solid gray; } 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739); 
} </string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zero">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	border: 1px solid gray; 
	background-color: rgb(243, 243, 243);
	color: black;
} 
QPushButton:pressed { 
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa); 
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
